### ISSUE TITLE
Text display detail bug?

### ISSUE THREAD

**Issue by neruthes** [tramhao/termusic#57]
The status bar looks like:

```
Playlist ──┤ Total 1 tracks | 03:30 | Loop: consume | Add:last
```

I believe that `Add:last` should be `Add: last` instead.

Version: termusic v0.6.11 (built from latest `dev` branch, 74fd4a0ea146cdafac0e39fb1f80b1498d8ec280)

**Comment by tramhao** [comment]
You mean the space in between? Thanks.

**Comment by tramhao** [comment]
Should be fixed in 64e443450d0cb8fdbd6746b187ef37d9b4618f0c of dev branch.

**Comment by neruthes** [comment]
lgtm
